ops/charm.py

```diff
@@ -1100,7 +1100,10 @@ class CollectStatusEvent(LifecycleEvent):
     way after the end of every hook, it should observe the
     :attr:`collect_app_status <CharmEvents.collect_app_status>` or
     :attr:`collect_unit_status <CharmEvents.collect_unit_status>` event,
-    respectively.
+    respectively. Note that observing the specific hook event is not required. For example, even
```

Comment:
This seems a bit confusing to me, in part because it's unclear what "the specific hook event" is referring to. I also don't think it needs to be in this first paragraph.

I wonder if we could make it even more succinct, and drop the example (hopefully we can make the language clear enough we don't need an example for this?). What about tweaking the paragraph below like so:

> The framework will trigger these events after the hook code runs
> successfully, on every Juju event whether it's `observe`d or not
> (``collect_app_status`` will only be triggered on the leader unit). ...

Reply:
Or as a new sentence:

> The framework will trigger these events after the hook code runs
> successfully (``collect_app_status`` will only be triggered on the leader
> unit). **This happens on every Juju event whether it was `observe`d or not.**

Reply:
Sounds good, that's very concise and clear